config/pipeline.yaml

```diff
@@ -622,6 +622,96 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.iommu
 
+  preempt-rt: &preempt-rt
```

Comment:
Is this a standard test suite like `kunit` or `fluster`?
I would need to add its info to KCIDB catalogue.
Please provide me with its details such as https://github.com/kernelci/kcidb/blob/main/tests.yaml#L70

Reply:
If the suite is `kernelci` specific, we should be using `kernelci_preempt_rt` as `kcidb_test_suite` property instead.

Reply:
We should go for a generic name to enable aggregation of our test results with other CI systems.
If the standard name is `rt-tests`, it's time to rename all the preempt-rt tests.
Should I use `rt-tests` as test suite name in KCIDB?
And yes, I can send in a PR for KCIDB test catalogue update. I sent its link just for reference.

Reply:
Yeah, let's use rt-tests name. I've updated the PRs.